coriolis/tests/api/v1/test_endpoint_destination_options.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from coriolis.api.v1 import endpoint_destination_options as endpoint
+from coriolis.api.v1.views import endpoint_options_view
+from coriolis.endpoint_options import api
+from coriolis.tests import test_base
+from coriolis import utils
+
+
+class EndpointDestinationOptionsControllerTestCase(
+    test_base.CoriolisBaseTestCase
+):
+    "Test suite for the Coriolis Endpoint Destination Options v1 API"""
```

Comment:
Turn it into a docstring:
```suggestion
    """Test suite for the Coriolis Endpoint Destination Options v1 API"""
```